recipes/qgis-plugin-manager/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "qgis-plugin-manager" %}
+{% set version = "1.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/qgis-plugin-manager-{{ version }}.tar.gz
+  sha256: 0bd36384d601df1a9b10b07ba8a4b9986001fa8ef531e66cb8cf9567bc8b65e2
+
+build:
+  entry_points:
+    - qgis-plugin-manager = qgis_plugin_manager.__main__:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+
+test:
+  imports:
+    - qgis_plugin_manager
+  commands:
+    - pip check
+    - qgis-plugin-manager --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/3liz/qgis-plugin-manager
+  summary: Tool for downloading/managing QGIS plugins from CLI.
+  dev_url: https://github.com/3liz/qgis-plugin-manager
+  license: 'GPL-3.0-only'
```

Comment:
Hi Yuvi, I see this file is in a PR upstream in https://github.com/3liz/qgis-plugin-manager/pull/61, right?
We need that to be merged first so we know upstream authors are OK with it before merging this PR.

Reply:
@ocefpaf thanks! I'll re-mark this as draft until that gets merged!

Reply:
I just merged the PR upstream. Thanks for this work.